src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/AdvSimdHelper.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+
+namespace System.Text.Encodings.Web
+{
+    internal static class AdvSimdHelper
```

Comment:
>AdvSimdHelper [](start = 26, length = 13)

I am wondering if it will be easier to have this file under `System.Private.CoreLib` (the portion that doesn't get crossgen) and source link it in various projects so all the .dlls can share the common implementation.